bottom/unpack.py

```diff
@@ -160,7 +160,8 @@
     ("485", "ERR_UNIQOPPRIVSNEEDED"),
     ("491", "ERR_NOOPERHOST"),
     ("501", "ERR_UMODEUNKNOWNFLAG"),
-    ("502", "ERR_USERSDONTMATCH")
+    ("502", "ERR_USERSDONTMATCH"),
+    ("TOPIC", "TOPIC")
```

Comment:
Just remembered why this line isn't needed: synonym lookup falls back to the normalized input value:

```python
def synonym(command: str) -> str:
    command = command.upper()
    return _2812_synonyms.get(command, command)
```

(yep, more cruft to burn away)